server/__init__.py

```diff
@@ -248,7 +257,9 @@ def create_app():
   app.config['BABEL_TRANSLATION_DIRECTORIES'] = 'i18n'
 
   # load disaster dashboard data from GCS
-  if os.environ.get('FLASK_ENV') in ['autopush', 'local', 'dev']:
+  if os.environ.get('FLASK_ENV') in [
+      'autopush', 'local', 'dev', 'local-stanford'
```

Comment:
should we also be including 'stanford'?

Reply:
Based on https://github.com/datacommonsorg/website/blob/b943707aeb967bba4a2377e54e736caca36fe25f/run_server.sh#L55, there is no 'stanford' FLASK_ENV.

Reply:
I believe this is the full list of flask environments: https://github.com/datacommonsorg/website/blob/master/server/lib/config.py#L29

Reply:
Thanks for linking! Added stanford